mlrun/feature_store/steps.py

```diff
@@ -276,7 +276,7 @@ def __init__(
         self.method = method
         self.default_value = default_value
 
-    def _impute(self, feature: str, value):
+    def _impute(self, feature: str, value: Any):
```

Comment:
how Any? if you are checking `pd.isna` over it?

Reply:
It can be `str`, `float`, `int`, `timestamp`, and a lot more (including None values), it depends on the data this feature holds, I am checking here `pd.isna` because it is checking all possible kinds of `None` values to replace in the imputing process. This is the same reason that the mapping class attribute is `Dict[str, Any]`.